lib/dialects/abstract/query.js

```diff
@@ -320,6 +320,41 @@ class AbstractQuery {
     return this.sql.toLowerCase().startsWith('call');
   }
 
+  /**
+   * @param {string} sql
+   * @param {Function} debugContext
+   * @protected
+   * @returns {Function} A function to call after the query was completed.
+   */
+  _logQuery(sql, debugContext) {
+    const { connection, options } = this;
+    const benchmark = this.sequelize.options.benchmark || options.benchmark;
+    const startTime = Date.now();
+    const fmt = `(${connection.uuid || 'default'}): ${sql}`;
+    const msg = `Executing ${fmt}`;
+    debugContext(msg);
+    if (!benchmark) {
+      this.sequelize.log(`Executing ${fmt}`, options);
+    }
+    return () => {
+      const afterMsg = `Executed ${fmt}`;
+      debugContext(afterMsg);
+      if (benchmark) {
+        this.sequelize.log(afterMsg, Date.now() - startTime, options);
+      }
+    };
+  }
+
+  /**
+   * Gets a debug context for sub classes.
+   *
+   * @param {string} dialect
+   * @private
+   */
+  static makeDebugContext(dialect) {
```

Comment:
Get rid of this method, let dialect query class create debug context by old `logger.debugContext(...)` method

Reply:
Resolved? `AbstractQuery` shouldn't be responsible for this, let's got with simple approach?

Reply:
Oh, final push didn't go through 😅